app/layout.tsx

```diff
@@ -0,0 +1,66 @@
+import "./globals.css";
+import { SideBar } from "@/components/sidebar/sidebar";
+
+// here I imported the font I want to use, even with this method I can use other fonts in other parts in the same way
+import { inter } from "@/types/fonts";
+export const metadata = {
+  title: "Poke Checker Made by Panda",
+};
+
+//<a href="/types/normal" className="w-0">types</a>
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body className={inter.className}>
+        <main className="mx-[2%] my-[2%]">
```

Comment:
Next.js 會直接將內容注入在 body 裡面，所以這邊不需要加上 html 和 body 標籤
```suggestion
<main className={`mx-[2%] my-[2%] ${inter.className}`}>
```

Reply:
豪ㄉ
resolved

Reply:
若layout沒有<body> and <html> tag
會出現
Uncaught Error: invariant expected app router to be mounted
ref: https://stackoverflow.com/questions/74515807/uncaught-error-invariant-expected-app-router-to-be-mounted